### PR TITLE
removed LiveLeak ( now defunct )

### PR DESCRIPTION
**Pull request by cronlabspl**

### CODE REVIEW

**Comment by Stefan-MyBB** [comment]
What about the actual MyCode parser?

**Comment by Stefan-MyBB** [comment]
There are more parts related to liveleak (e.g. template, language strings, help doc).

File: admin/inc/functions_themes.php [hunk 1]
```diff
@@ -6,9 +6,6 @@
  * Website: http://www.mybb.com
  * License: http://www.mybb.com/about/license
  *
- */
```
Comment (Stefan-MyBB) [inline admin/inc/functions_themes.php:9]: Why are you changing this?

File: admin/inc/functions_themes.php [hunk 2]
```diff
@@ -1510,6 +1479,7 @@ function upgrade_css_120_to_140($css)
 		}
 	}
 
+	// MD5 is not the safest method to use.
```
Comment (Stefan-MyBB) [inline admin/inc/functions_themes.php:1482]: What is the point of this comment? These are not passwords so md5 is perfectly fine here.

File: install/resources/mybb_theme.xml [hunk 3]
```diff
@@ -13773,7 +13773,7 @@ if(use_xmlhttprequest == "1")
 		<template name="video_metacafe_embed" version="1827"><![CDATA[<iframe src="http://www.metacafe.com/embed/{$id}/" width="440" height="248" allowfullscreen="true" frameborder=0></iframe>]]></template>
 		<template name="video_dailymotion_embed" version="1827"><![CDATA[<iframe frameborder="0" width="480" height="270" src="//www.dailymotion.com/embed/video/{$id}" allowfullscreen="true"></iframe>]]></template>
 		<template name="video_facebook_embed" version="1800"><![CDATA[<iframe src="https://www.facebook.com/video/embed?video_id={$id}" width="625" height="350" frameborder="0"></iframe>]]></template>
-		<template name="video_liveleak_embed" version="1827"><![CDATA[<iframe width="500" height="300" src="https://www.liveleak.com/ll_embed?i={$id}" frameborder="0" allowfullscreen="true"></iframe>]]></template>
+
```
Comment (Stefan-MyBB) [inline install/resources/mybb_theme.xml]: Can you remove the empty line please?

**Review by Stefan-MyBB — CHANGES_REQUESTED** [review]
You've included some changes not related to Liveleak.

File: admin/inc/functions_themes.php (hunk 1 above)
Comment (cronlabspl) [reply admin/inc/functions_themes.php:9]: Why not? Looks better IMHO

File: admin/inc/functions_themes.php (hunk 2 above)
Comment (cronlabspl) [reply admin/inc/functions_themes.php:1482]: I know these are not passwords;  But still, its not safe to use MD5 as its [easily unhashable](https://www.google.com/search?client=safari&rls=en&q=unhash+md5&ie=UTF-8&oe=UTF-8)

File: install/resources/mybb_theme.xml (hunk 3 above)
Comment (cronlabspl) [reply install/resources/mybb_theme.xml:13776]: Yes; sure

File: admin/inc/functions_themes.php (hunk 1 above)
Comment (Stefan-MyBB) [reply admin/inc/functions_themes.php:9]: Both comments serve entirely different purposes, so it makes very little sense to merge them.

File: admin/inc/functions_themes.php [hunk 4]
```diff
@@ -502,24 +499,6 @@ function resync_stylesheet($stylesheet)
 	return false;
 }
 
-/**
```
Comment (Stefan-MyBB) [inline admin/inc/functions_themes.php:505]: I thought you accidentally included this. Removing functions in a minor point release may introduce compatibility issues.

File: admin/inc/functions_themes.php [hunk 5]
```diff
@@ -530,16 +509,6 @@ function fix_css_urls_callback($matches)
 	return fix_css_urls($matches[2]);
 }
 
-/**
```
Comment (Stefan-MyBB) [inline admin/inc/functions_themes.php:533]: Same as above.

**Pull request closed without merging**